packages/react-core/src/components/Pagination/Navigation.tsx

```diff
@@ -29,7 +29,7 @@ export interface NavigationProps extends React.HTMLProps<HTMLElement> {
   /** Label for the English word "of". */
   ofWord?: string;
   /** The number of the current page. */
-  page: string | number;
+  page: number;
```

Comment:
lgtm overall but cc @tlabaj to double check that this prop change isn't also a breaking change (while `onPageInput` was a type fix).

Reply:
This interface is never exported as a public API, so this will not be a breaking change.